rust-lisa/src/main/java/it/unipr/cfg/statement/RustUnsafeExitStatement.java

```diff
@@ -0,0 +1,63 @@
+package it.unipr.cfg.statement;
+
+import it.unive.lisa.analysis.AbstractState;
+import it.unive.lisa.analysis.AnalysisState;
+import it.unive.lisa.analysis.SemanticException;
+import it.unive.lisa.analysis.StatementStore;
+import it.unive.lisa.analysis.heap.HeapDomain;
+import it.unive.lisa.analysis.value.TypeDomain;
+import it.unive.lisa.analysis.value.ValueDomain;
+import it.unive.lisa.interprocedural.InterproceduralAnalysis;
+import it.unive.lisa.program.cfg.CFG;
+import it.unive.lisa.program.cfg.CodeLocation;
+import it.unive.lisa.program.cfg.edge.Edge;
+import it.unive.lisa.program.cfg.statement.Statement;
+import it.unive.lisa.util.datastructures.graph.GraphVisitor;
+
+/**
+ * Rust unsafe exiting statement.
+ * 
+ * @author <a href="mailto:vincenzo.arceri@unipr.it">Vincenzo Arceri</a>
+ * @author <a href="mailto:simone.gazza@studenti.unipr.it">Simone Gazza</a>
+ */
+public class RustUnsafeExitStatement extends Statement {
+
```

Comment:
Missing reference to its `RustUnsafeEntryStatement`?

Reply:
Fixed in 4b5061a.